doc/explorers/explore.ipynb

```diff
@@ -1,9 +1,25 @@
 {
  "cells": [
   {
+   "cell_type": "markdown",
```

Comment:
This file appears to be identical to `browsing_cubes/explore.ipynb` above. However, I cannot get this notebook to plot anything...